helpers/training/validation.py

```diff
@@ -968,6 +968,8 @@ def setup_pipeline(self, validation_type, enable_ema_model: bool = True):
                 "tokenizer": self.tokenizer_1,
                 "vae": self.vae,
                 "safety_checker": None,
+                "text_encoder_2": None,
+                "text_encoder_3": None,
             }
```

Comment:
```suggestion
                "tokenizer": self.tokenizer_1,
                "vae": self.vae,
                "safety_checker": None,
            }
            if self.args.model_family in ["sd3", "sdxl", "flux"]:
                extra_pipeline_kwargs["text_encoder_2"] = None
            if self.args.model_family in ["sd3"]:
                extra_pipeline_kwargs["text_encoder_3"] = None
```

Reply:
Yeah, that makes sense. I added your changes to the PR.